content/en/blog/_posts/2025-04-29-hpa-configurable-tolerance.md

```diff
@@ -1,9 +1,9 @@
 ---
 layout: blog
 title: "Kubernetes v1.33: HorizontalPodAutoscaler Configurable Tolerance"
-slug: kubernetes-1-33-hpa-configurable-tolerance
+slug: kubernetes-v1-33-hpa-configurable-tolerance
 math: true # for formulae
-date: 2025-04-29T10:30:00-08:00
+date: 2025-04-28T10:30:00-08:00
```

Comment:
file name still references April 29 `content/en/blog/_posts/2025-04-29-hpa-configurable-tolerance.md`, should we update?

Reply:
Oh good point, I need to update the file  name